packages/cli/src/commands/import.ts

```diff
@@ -0,0 +1,60 @@
+import { Command, flags } from '@oclif/command';
+import { promises as fs } from 'fs';
+import { Messages } from '../constants/messages.constants';
+import { parseDataFiles } from '../libs/data';
+
+export default class Import extends Command {
+  public static description = 'Import a mock API';
+  public static examples = [
+    '$ mockoon-cli import --data ~/data.json --output ./output.json',
+    '$ mockoon-cli import --data ~/data.json --output ./output.json --type open-api-v3'
+  ];
+
+  public static flags = {
+    data: flags.string({
+      char: 'd',
+      description: 'Path or URL to your data file',
+      required: true
+    }),
+    output: flags.string({
+      char: 'o',
+      description:
+        'Generated Mockoon path and name (e.g. `./environment.json`)',
+      required: true
+    }),
+    port: flags.integer({
+      char: 'p',
+      description: 'Environment port',
+      default: 8080
+    }),
```

Comment:
I am not sure having a port flag here is relevant as we can already override the port at runtime.

Reply:
It was just an additional feature that I was thinking about adding, but it does not bring much gain.
I'm going to remove it to keep this command with the same export behavior in the desktop app.